test/end-to-end.js

```diff
@@ -19,6 +19,7 @@ var TestDataBuilder = require('./helpers/test-data-builder');
 var ref = TestDataBuilder.ref;
 var given = require('./helpers/given');
 var should = require('chai').should();
+var platform = require('./helpers/platform');
```

Comment:
Perhaps `var isWindows = require('./helpers/platform').isWindows` to make the test code below shorter and more likely to fit within 80 chars?